CRUD/AddCart/Api/DataInterface.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
@msakila532 Module namespace should be of Brand or Client business name. It shouldn't be related to module functionality. You've to utilize module name for that